Vacuum/Clean.fs

```diff
@@ -173,27 +178,33 @@ let clean({
     info $"Cleaning directory %s{directory.RawPathString}"
     if bytesToFree.IsSome then
         info $"Cleaning %d{bytesToFree.Value} bytes"
+    elif freeUntil.IsSome then
+        info $"Cleaning until %d{freeUntil.Value} bytes are free"
 
```

Comment:
My suggestion would be to convert `freeUntil` into `bytesToFree` here. Something among the lines:
```fsharp
// Check for contradictory parameter values
if bytesToFree.IsSome && freeUntil.IsSome then failwithf "Incorrect parameter combination: both bytesToFree and freeUntil are passed"
let bytesToFree = 
    if bytesToFree.IsSome then bytesToFree
    elif freeUntil.IsSome
        let freeDiskSpace = DriveInfo(directory).AvailableFreeSpace
        let bytesToClean = max 0 (freeUntil.Value - freeDiskSpace)
        Some bytesToClean
```

This will allow you to implement the new feature without touching any of the existing cleanup logic (and it's good enough for me).

What do you think? I can finish if myself of course.